html/about.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="../css/main.css">
+    <link rel="stylesheet" href="../css/about.css">
+    <script src="../js/about.js" type="module" async></script>
+    <title>About - Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
+    <main id="main" class="container-fullwidth">
+      <div>
+        <nav id="nav" class="container-fullwidth navbar">

```

Comment:
- [ ] Great job creating the about page, let's add the missing menu bar in the desktop version:

Template:
![Captura de pantalla 2022-04-14 a la(s) 8 02 22 p  m](https://user-images.githubusercontent.com/83717285/163501671-522556bd-667d-443b-8a2d-e3d669c736ff.png)


Your design:
![Captura de pantalla 2022-04-14 a la(s) 8 02 38 p  m](https://user-images.githubusercontent.com/83717285/163501629-0017f826-82d3-4e31-9af1-80681d6c874c.png)